### PR TITLE
Ernie feature setup

### PR DESCRIPTION
**Pull request by Ernie1234**
backend folders and files setup for backend

### CODE REVIEW

**Comment by Ernie1234** [comment]
backend setup and eslint

**Review by greatertomi — CHANGES_REQUESTED** [review]
Good initial setup. Please add [Airbnb rules](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) don't forget to add [plugin with typescript](https://github.com/iamturns/eslint-config-airbnb-typescript) support. Also add [unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn) and [sonarjs](https://github.com/SonarSource/eslint-plugin-sonarjs), I find them very helpful. Incase you are interested, you can find more awesome eslint plugins [here](https://github.com/dustinspecker/awesome-eslint).

Btw, if you want to add any rules apart from the ones I specify above, make sure you inform me.

**Comment by greatertomi** [comment]
Also, your branch is not named properly. Read [this](https://medium.com/@abhay.pixolo/naming-conventions-for-git-branches-a-cheatsheet-8549feca2534). Btw, don't rename this branch, just note that for next time.

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import express, { Request, Response } from "express";
+import dotenv from "dotenv";
+import cors from "cors";
+
+//For env File
```
[the remark this answers is not in the document]
Comment (Ernie1234) [reply src/index.ts:5]: Okay sir, noted sir!

File: package.json [hunk 2]
```diff
@@ -4,9 +4,29 @@
...
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.4.5",
+    "express": "^4.19.2",
+    "mongodb": "^6.6.2",
+    "mongoose": "^8.4.0",
+    "nodemon": "^3.1.1"
```
[the remark this answers is not in the document]
Comment (Ernie1234) [reply package.json:21]: okay sir, done!

**Pull request closed without merging**